com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -32,6 +32,11 @@ public class DemonstrationRecorder : MonoBehaviour
         [Tooltip("Whether or not to record demonstrations.")]
         public bool Record;
 
+        /// <summary>
+        /// Number of steps to record. Editor will stop playing when it hits this threshold.
```

Comment:
Can you add this as a `[Tooltip(...)]` too?

Reply:
added tooltip